lib/htmldiff.rb

```diff
@@ -136,33 +137,9 @@ def find_match(start_in_old, end_in_old, start_in_new, end_in_new)
         match_length_at = new_match_length_at
       end
 
-#      best_match_in_old, best_match_in_new, best_match_size = add_matching_words_left(
-#          best_match_in_old, best_match_in_new, best_match_size, start_in_old, start_in_new)
-#      best_match_in_old, best_match_in_new, match_size = add_matching_words_right(
-#          best_match_in_old, best_match_in_new, best_match_size, end_in_old, end_in_new)
-
       return (best_match_size != 0 ? Match.new(best_match_in_old, best_match_in_new, best_match_size) : nil)
     end
 
-    def add_matching_words_left(match_in_old, match_in_new, match_size, start_in_old, start_in_new)
```

Comment:
What happened here? Is it not needed?

Reply:
Exactly! It was only called from the commented out lines above. Since it's been this way for 9 years, i deleted the code. Otherwise I have tried to avoid cleanups.